services/app-api/forms/sar.json

```diff
@@ -18,8 +18,12 @@
         },
         "praDisclosure": [
           {
-            "type": "span",
-            "content": "<b>PRA Disclosure Statement</b><br/>Under the Privacy Act of 1974 any personally identifying information obtained will be kept private to the extent of the law. According to the Paperwork Reduction Act of 1995, no persons are required to respond to a collection of information unless it displays a valid OMB control number. The valid OMB control number for this information collection is 0938-1053. The time required to complete this information collection is estimated to average 2.5 hours per response, including the time to review instructions, search existing data resources, gather the data needed, and complete and review the information collection. If you have comments concerning the accuracy of the time estimate(s) or suggestions for improving this form, please write to: CMS, 7500 Security Boulevard, Attn: PRA Reports Clearance Officer, Mail Stop C4-26-05, Baltimore, Maryland 21244-1850"
+            "type": "p",
```

Comment:
Semantically, this should probably be a heading, but that might be a can of worms.

Reply:
Agreed...I'm hoping maybe we can figure it out as part of our semantic structure spike?